edgelet/build/windows/build-diagnostics.ps1

```diff
@@ -4,11 +4,23 @@
  # Builds and publishes to target/publish/ the iotedge-diagnostics binary and its associated dockerfile
  #>
 
+param(
+    [ValidateSet("debug", "release")]
+    [string]$BuildConfiguration = "release"
+)
+ 
 $ErrorActionPreference = 'Continue'
 
 . (Join-Path $PSScriptRoot 'util.ps1')
 
 Assert-Rust
+If ($BuildConfiguration.ToLower() == "release") {
```

Comment:
`if ($BuildConfiguration -eq 'release') {`

PS does not have `==`, and `-eq` is already case-insensitive.